t/plugin/limit-conn2.t

```diff
@@ -321,8 +351,8 @@ request latency is nil
                  [[{
                         "plugins": {
                             "limit-conn": {
-                                "conn": 2,
-                                "burst": 1,
+                                "conn": 1,
```

Comment:
Why change this configuration?

Reply:
It is easier to test when value of key is empty.

Reply:
Why changing the configuration doesn't provide a different result?

Reply:
I see. It is already covered in the new test.